config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   root to: "pages#home"
 
+  authenticate :user, ->(user) { user.admin? } do
+    mount GoodJob::Engine => "good_job"
+  end
```

Comment:
My understanding is that GoodJob is used to manage the background jobs, like emailing.
So in this block of code I understand that the program mounts the GoobJob engine only if the user is the admin.
Do we want to launch the background jobs in a queue only when the admin says 'go' in some way ?

Reply:
This the routes folder
It means this is a path to some url endpoint.
In this case it's wesbite/good_job

What this all means is that only the admin can access this page

Reply:
It's a dashboard provided by the gem to monitor background tasks
![image](https://github.com/wJoenn/YogaUnalome/assets/75388869/decb8d43-cb72-46f9-8088-f70919287ed4)